src/vault/policy-roots/impl/system-policy.service.ts

```diff
@@ -47,38 +49,30 @@ export class SystemPolicyService implements PolicyRootService<undefined> {
    */
   public async buildSystem(): Promise<HlcRenderSpec[]> {
     this.logger.debug(`Build system - global`);
-    return [
-      {
-        group: VAULT_ROOT_SYSTEM,
-        templateName: 'admin-super',
-        data: {
-          kvPaths: await this.config.getKvStores(),
-        },
-      },
-      { group: VAULT_ROOT_SYSTEM, templateName: 'admin-general' },
-      { group: VAULT_ROOT_SYSTEM, templateName: 'admin-token' },
-      { group: VAULT_ROOT_SYSTEM, templateName: 'admin-audit-hash' },
-      {
-        group: VAULT_ROOT_SYSTEM,
-        templateName: 'broker-auth',
-        data: {
-          authMount: VAULT_APPROLE_MOUNT_POINT,
-          restrictedPaths: await this.restrictedBrokerAppPaths(),
-        },
-      },
-      {
-        group: VAULT_ROOT_SYSTEM,
-        templateName: 'db-admin-super',
-        data: { secertDbPath: 'db' },
-      },
-      { group: VAULT_ROOT_SYSTEM, templateName: 'isss-cdua-read' },
-      { group: VAULT_ROOT_SYSTEM, templateName: 'isss-ci-read' },
-      { group: VAULT_ROOT_SYSTEM, templateName: 'nrcd-read' },
-      { group: VAULT_ROOT_SYSTEM, templateName: 'oraapp-imborapp-read' },
-      { group: VAULT_ROOT_SYSTEM, templateName: 'user-generic' },
-      { group: VAULT_ROOT_SYSTEM, templateName: 'vault-sync' },
-    ];
+    const sysSpecs: HlcRenderSpec[] = [];
+    // Directory containing the template files
+    const templatesDir = path.join(__dirname, '../../../../config/system');
```

Comment:
static value. Does not need to be recomputed each time buildSystem is called. see: ConfigFileService.policyFilePath